### PR TITLE
Test the edit, checkbox and Clear functionalities

### PR DESCRIPTION
**Pull request by trace-kadenyi**
- Implement three tests to verify the functionalities for the:
        - Edit function
        - Checkbox function
        - Clear All Completed function

### CODE REVIEW

File: src/test/todoObjectClass.test.js [hunk 1]
```diff
@@ -24,24 +25,59 @@ describe('add and remove tasks', () => {
...
+  test('clear all completed', () => {
+    let myArray = [
+      { index: 1, description: 'Watch a movie', completed: false },
+      { index: 2, description: 'Wash my clothes', completed: false },
+      { index: 3, description: 'Visit a friend', completed: true },
+    ];
+    myArray = myArray.filter((task) => task.completed === false);
+
+    expect(myArray).toHaveLength(2);
+  });
```
Comment (julie-ify) [inline src/test/todoObjectClass.test.js]: - [x] Good job so far, however, please kindly make use of the function which you created for clearing all completed tasks to test the clear all completed functionality. Tests are written to test our functions to ensure they work as expected. So, please test the `Clear all completed` function which you called `clearSelectedTasks` in your index.js.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @TracK92 and @gak92,

Good job so far!

## Highlights!
- GitHub Flow was followed ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- tests passed ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/todoObjectClass.test.js (hunk 1 above)
Comment (trace-kadenyi) [reply src/test/todoObjectClass.test.js]: Hi there, 

I have created a function for that in the todoObjectClass.js called `clearTasks` and called it in both the index.js and the test. Kindly, re-review my work.

**Review by Mre55 — APPROVED** [review]
**Hi @TracK92,**

# Status: APPROVED 🏆 🎉

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- All test written well and pass ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

Alt Text


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**